juno-migration/9822622/project_juno.yaml

```diff
@@ -0,0 +1,107 @@
+specVersion: 1.0.0
+name: juno-dictionary
+version: 1.0.0
+runner:
+  node:
+    name: "@subql/node-cosmos"
+    version: ">=3.0.0"
+    options:
+      unfinalizedBlocks: false
+      historical: false
+  query:
+    name: "@subql/query"
+    version: "*"
+description: A SubQuery project provide dictionary data & service for the Terra network
+schema:
+  file: ./schema.graphql
+
+parent:
+  reference: QmPjq55mgUt9S8S491Q3wEbb87fXyEkdxymT6Gwe2xe1Z1
```

Comment:
This should be `QmSNksgvSNHc5YCrfu63pmzoxpdzVciUFwgvRdk118dejs`

`2xe1Z1` is same as in 9817051 yaml

Reply:
Just remove directory. Here fixed in 2 blocks.
**this is caused by a very long response time to a pull request**

**DONE**